meeshkan/serve/mock/faker/stateless_faker.py

```diff
@@ -0,0 +1,323 @@
+            else False
+        )
+
+    # TODO: add exclusiveMinimum and exclusiveMaximum
+    def fake_integer(self, schema: Any) -> int:
+        mn = self._LO if "minimum" not in schema else schema["minimum"]
+        mx = self._HI if "maximum" not in schema else schema["maximum"]
+        return (
+            random.choice(schema["enum"])
+            if "enum" in schema
+            else random.randint(mn, mx)
+        )
+
+    def fake_ref(self, faker_data: FakerData, schema: Any, depth: int):
+        name = schema["$ref"].split("/")[2]
+        return self.fake_it(faker_data.schema["definitions"][name], depth)
+
+    def fake_ref_array(
+        self, faker_data: FakerData, schema: Any, depth: int, count: int
+    ):
+        name = schema["$ref"].split("/")[2]
+        return [
+            self.fake_it(faker_data.schema["definitions"][name], depth)
+            for _ in range(count)
+        ]
+
+    def fake_null(self, schema: Any) -> None:
+        return None
+
+    def fake_number(self, schema: Any) -> float:
+        mn = self._LO if "minimum" not in schema else schema["minimum"]
+        mx = self._HI if "maximum" not in schema else schema["maximum"]
+        return (
+            random.choice(schema["enum"])
+            if "enum" in schema
+            else (random.random() * (mx - mn)) + mn
+        )
+
+    def fake_it(self, faker_data: FakerData, schema: Any, depth: int) -> Any:
+        depth += 1
+        return (
+            self.fake_array(faker_data, schema, depth)
+            if ("type" in schema) and (schema["type"] == "array")
+            else self.fake_any_of(faker_data, schema, depth)
+            if "anyOf" in schema
+            else self.fake_all_of(faker_data, schema, depth)
+            if "allOf" in schema
+            else self.fake_one_of(faker_data, schema, depth)
+            if "oneOf" in schema
+            else self.fake_not(faker_data, schema, depth)
+            if "not" in schema
+            else self.fake_ref(faker_data, schema, depth)
+            if "$ref" in schema
+            else self.fake_object(faker_data, schema, depth)
+            if ("type" not in schema) or (schema["type"] == "object")
+            else self.fake_string(schema)
+            if schema["type"] == "string"
+            else self.fake_integer(schema)
+            if schema["type"] == "integer"
+            else self.fake_boolean(schema)
+            if schema["type"] == "boolean"
+            else self.fake_null(schema)
+            if schema["type"] == "null"
+            else self.fake_number(schema)
+            if schema["type"] == "number"
+            else {}
+        )
+
+    def process(self, spec: OpenAPIObject, storage: MockData, request: Request) -> Any:
+        path_item, path_candidate = random.choice([x for x in spec.paths.items()])
+
+        entity_name = get_x(path_candidate, "x-meeshkan-entity")
+        entity = storage.get_entity(entity_name) if entity_name is not None else None
```

Comment:
Why does stateless faker need to care about entities?

Reply:
Removed it. And I removed the default faker because right now the StatefulFaker preserves the behaviour of the StatelessFaker. So everything related to storages/entities is removed from both the BaseFaker and the StatelessFaker.